helmfile/01-applications.yaml

```diff
@@ -162,6 +162,23 @@ releases:
   values:
   - values/calico-accountant.yaml.gotmpl
 
+
+# cluster-admin-rbac
+- name: cluster-admin-rbac
+  namespace: kube-system
+  labels:
+    app: cluster-admin-rbac
+  chart: ./charts/cluster-admin-rbac
+  version: 0.1.0
+  missingFileHandler: Error
+  {{ if .Values.opa.enabled }}
+  needs:
+  - gatekeeper-system/gatekeeper-constraints
+  {{ end }}
```

Comment:
```suggestion
  {{ if eq .Environment.Name "workload_cluster" }}
  {{ if .Values.opa.enabled }}
  needs:
  - gatekeeper-system/gatekeeper-constraints
  {{ end }}
  {{ end }}
```
This seems to be required in the service cluster

Reply:
Why does the chart need gatekeeper-constraints?

Reply:
I wasn't sure about this - I just included it since `user-rbac` includes it. Does `user-rbac` only need it because it creates some extra namespaces?

Reply:
Right, I didn't even consider if it was needed. Yes user-rbac should only include it because of the namespaces it creates. So you should be able to delete it here.

Reply:
```suggestion
```
This whole part can be removed.

Reply:
My bad, completely forgot about this.